### PR TITLE
Fields order is kept from the source

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: django_geosource/models.py [hunk 1]
```diff
@@ -162,7 +163,9 @@ def update_fields(self):
 
                 if field_name not in fields:
                     field, is_new = self.fields.get_or_create(
-                        name=field_name, defaults={"label": field_name}
+                        name=field_name,
+                        defaults={"label": field_name},
+                        order=len(fields.items()) + 1,
```
Comment (jrmi) [inline django_geosource/models.py:168]: Use enumerate instead

Comment (CFarcy) [reply django_geosource/models.py:168]: Yes, you're right. I didn't know you could enumerate on a dict (even if it makes absolute sense in python the more I think about it :D).

Comment (CFarcy) [reply django_geosource/models.py:168]: I'm not sure to understand, we are not "looping" through `fields`

Comment (jrmi) [reply django_geosource/models.py:168]: Yeah but you loop through record.items() i think ?

Comment (CFarcy) [reply django_geosource/models.py]: Done.

**Review by jrmi — APPROVED** [review]

**Pull request merged**